src/lib/types.ts

```
@@ -135,29 +136,31 @@ export interface LinkType {
 
-interface ButtonAttibutes extends Omit<HTMLButtonAttributes, "on:copy" | "oncopy"> {
-  href?: undefined;
-}
+export type AnchorDivAttributes =
+  | ({ href: string } & HTMLAnchorAttributes)
+  | ({ href?: undefined } & HTMLAttributes<HTMLDivElement>);
 
-export type AnchorButtonAttributes = AnchorAttributes | ButtonAttibutes;
+export type AnchorButtonDivAttributes =
+  | ({ href: string } & HTMLAnchorAttributes)
+  | ({ href?: undefined, onclick: Function } & HTMLButtonAttributes)
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)